sandboxes/cuckoo.py

```diff
@@ -0,0 +1,36 @@
+import io
+import logging
+import requests
+from common import parse_config
+conf = parse_config()
+
+logger = logging.getLogger('pastehunter')
+
+def upload_file(raw_file, paste_object):
+    try:
+        task_id = send_to_cuckoo(raw_file, paste_object["pasteid"])
+        paste_object["Cuckoo Task ID"] = task_id
+        logger.info("exe submitted to Cuckoo with task id {0}".format(task_id))
+    except Exception as e:
+        logger.error("Unabled to submit sample to cuckoo")
+
+    # Send any updated json back
+    return paste_object
+
+def send_to_cuckoo(raw_exe, pasteid):
+    cuckoo_ip = conf["sandboxes"]["cuckoo"]["api_host"]
+    cuckoo_port = conf["sandboxes"]["cuckoo"]["api_port"]
+    cuckoo_host = 'http://{0}:{1}'.format(cuckoo_ip, cuckoo_port)
```

Comment:
Can we make this configurable and default to TLS? (assuming cuckoo isn't http-only)

Reply:
Before I make this change do you know why it's http in the first place? Is that the default and normal settings for Cukcoo out of the box. When I approached the change code wise my goal was not to change any of the existing behavior of the sandboxes to make it easier to tell that the refactor didn't break anything. Thus I didn't change anything here logic wise really.

If we do want to make a change would it make more sense to just make a `cuckoo_url` config parameter rather than a tls true false? Something like that would also affect backward compatibility with config stuff though.

Reply:
Hmmm, yeah, interesting question. That I'm also not sure about. Will wait for comments from @kevthehermit on this and the other compatibility Q. (If we don't get a response in a few days I think we should be fine to leave the PR as-is)